src/status_im/chat/models/message.cljs

```diff
@@ -165,7 +174,8 @@
                      (fx/merge cofx
                                (add-received-message message-with-chat-id)
                                (update-unviewed-count message-with-chat-id)
-                               (chat-model/join-time-messages-checked chat-id))))))))
+                               (chat-model/join-time-messages-checked chat-id)
+                               (check-for-incoming-tx message-with-chat-id))))))))
```

Comment:
why do we want to check tx for every incoming message?

Reply:
Some messages contain tx-hash and we need to add that tx to wallet history. What would you suggest otherwise?

Reply:
so it will check only if there is transaction in the message, ok then :)